src/styles/globals.css

```
@@ -32,30 +32,53 @@
 }
 
 body {
   background: var(--background);
   color: var(--foreground);
   font-family: var(--font-sans);
+  -webkit-font-smoothing: antialiased; /* Crisp text rendering in Safari/macOS */
+  -moz-osx-font-smoothing: grayscale;
+}
+
+/* WebKit Marquee Keyframes & Animation */
+@-webkit-keyframes marquee {
```

Stylelint (17.14.0)

[error] Vendor-prefixed at-rule "`@-webkit-keyframes`" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)